mgmt/rest/v2/config.go

```diff
@@ -33,17 +33,51 @@ type PolicyTable cpolicy.RuleTable
 
 type PolicyTableSlice []cpolicy.RuleTable
 
-// cdata.ConfigDataNode implements it's own UnmarshalJSON
 type PluginConfigItem struct {
 	cdata.ConfigDataNode
 }
 
+// PluginConfigResponse represents the response of a plugin config items.
+//
+// swagger:response PluginConfigResponse
+type PluginConfigResponse struct {
```

Comment:
Ok, so all `Response` structs are just for declaring swagger specification and generating client, now I understand. I was confused cause I was looking on `/plugins` call swagger spec:

```
// Responses:
// 200: PluginsResponse
```

and `getPlugins()` handler method:

```go
Write(200, PluginsResponse{Plugins: filteredPlugins}, w)
```

and in this case the response struct passed to `Write()` is the same as in swagger spec, but now I see that even in this case, the "real" PluginsResponse for generating client is actually `PluginsResp`:

```go
// PluginsResp represents the response from plugins operations.
//
// swagger:response PluginsResponse
type PluginsResp struct {
	// List of plugins
	//
	// in: body
	Body struct {
		Plugins []Plugin `json:"plugins,omitempty"`
	}
}
```